### PR TITLE
Change description of registration-district register

### PR DESCRIPTION
**Pull request by nacnudus**

### CODE REVIEW

File: data/alpha/register/registration-district.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ fields:
 phase: alpha
 register: registration-district
 registry: hm-passport-office
-text: Registration districts in England and Wales, defined for the purposes of the Registration Acts
+text: Registration districts in England and Wales, defined for the purposes of the Civil Registrations Acts
```
Comment (endamccormack) [inline data/alpha/register/registration-district.yaml]: I'm sorry, I know i'm being awfully pedantic about this lately but I think its OK the put in what you did in terms of code, but when you are changing data I think the history as to why is much more important.

i.e. `updated the description text to add clarity to which registration act we are referencing`

Comment (endamccormack) [inline data/alpha/register/registration-district.yaml]: This is not a you issue, we have not been doing this well traditionally and I would like to improve it.

**Comment by rachsuth** [comment]
The change to the register description has been requested by the register Custodian.  The new description provides more specific information about the relevant legislation.

File: data/alpha/register/registration-district.yaml (hunk 1 above)
Comment (nacnudus) [inline data/alpha/register/registration-district.yaml]: Good points, next time I'll include a snippet of the relevant emails.  Maybe we should use GitHub issues, but that becomes yet another channel to track.

Comment (endamccormack) [inline data/alpha/register/registration-district.yaml]: @nacnudus Im confused... why didn't you update the commit before merging this?

Comment (nacnudus) [inline data/alpha/register/registration-district.yaml]: @endamccormack I was too late, it had already been merged, along with a relevant comment below.

**Pull request merged**